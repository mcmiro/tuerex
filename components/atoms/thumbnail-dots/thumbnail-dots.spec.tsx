import * as React from "react";
import {screen, render} from "@testing-library/react";
import ThumbnailDots from "./";

describe("Test ThumbnailDots component", () => {
	it("Should pass when ThumbnailDots is defined", () => {
		render(<ThumbnailDots data-testid="thumbnail-dots-test" />);

		const thumbnailDots = screen.queryByTestId("thumbnail-dots-test");
		expect(thumbnailDots).toBeDefined();
		expect(thumbnailDots).toMatchSnapshot();
	});
});
