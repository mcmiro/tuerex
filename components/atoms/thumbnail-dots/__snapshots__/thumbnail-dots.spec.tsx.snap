// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ThumbnailDots component Should pass when ThumbnailDots is defined 1`] = `null`;
